releases/links_v2.0/LINKS

```diff
@@ -0,0 +1,119 @@
+                -o  offset position for extracting k-mer pairs (default -o 0, optional)\n
+                -e  error (%) allowed on -d distance   e.g. -e 0.1  == distance +/- 10% (default -e 0.1, optional)\n
+                -l  minimum number of links (k-mer pairs) to compute scaffold (default -l 5, optional)\n
+                -a  maximum link ratio between two best contig pairs (default -a 0.3, optional)\n
+                \t *higher values lead to least accurate scaffolding*\n
+                -z  minimum contig length to consider for scaffolding (default -z 500, optional)\n
+                -b  base name for your output files (optional)\n
+                -r  Bloom filter input file for sequences supplied in -s (optional, if none provided will output to .bloom)\n
+                \t NOTE: BLOOM FILTER MUST BE DERIVED FROM THE SAME FILE SUPPLIED IN -f WITH SAME -k VALUE\n
+                \t IF YOU DO NOT SUPPLY A BLOOM FILTER, ONE WILL BE CREATED (.bloom)\n
+                -p  Bloom filter false positive rate (default -p 0.001, optional; increase to prevent memory allocation errors)\n
+                -x  Turn off Bloom filter functionality (-x 1 = yes, default = no, optional)\n
+                -v  Runs in verbose mode (-v 1 = yes, default = no, optional)
+	";
+	}
+
+while getopts "f:s:m:d:k:t:j:o:e:l:a:z:b:r:p:x:v:" args; do
+    case $args in
+        f)
+            f=${OPTARG}
+	        f="draft=${f}"
+            ;;
+        s)
+            s=${OPTARG}
+	        s="readsFof=${s}"
+            ;;
+	    m)
+            m=${OPTARG}
+	        f="m=${m}"
+            ;;
+        d)
+            d=${OPTARG}
+	        d="d=${d}"
+            ;;
+        k)
+            k=${OPTARG}
+	        k="k=${k}"
+            ;;
+        t)
+            t=${OPTARG}
+	        t="t=${t}"
+            ;;
+	    o)
+            o=${OPTARG}
+	        o="o=${o}"
+            ;;
+        e)
+            e=${OPTARG}
+	        e="e=${e}"
+            ;;
+        l)
+            l=${OPTARG}
+	        l="l=${l}"
+            ;;
+        a)
+            a=${OPTARG}
+	        a="a=${a}"
+            ;;
+	    z)
+            z=${OPTARG}
+	        z="z=${z}"
+            ;;
+        b)
+            b=${OPTARG}
+	        b="b=${b}"
+            ;;
+        r)
+            r=${OPTARG}
+	        r="r=${r}"
+            ;;
+        p)
+            p=${OPTARG}
+	        p="p=${p}"
+            ;;
+	    x)
+            x=${OPTARG}
+	        x="x=${x}"
+            ;;
+        v)
+            v=${OPTARG}
+            v="v=${v}"
+            ;;
+        *)
+            usage
+            ;;
+    esac
+done
+shift $((OPTIND-1))
+
+# Check if draft genome and long read fof is provided
+if [ -z "${f}" ] || [ -z "${s}" ]; then
+    usage
+    exit 1
+fi
+
+echo Running Command: ./LINKS-make LINKS $f $s $m $d $k $t $o $e $l $a $z $b $r $p $x $v
+# Running LINKS-make
+./LINKS-make LINKS $f $s $m $d $k $t $o $e $l $a $z $b $r $p $x $v
```

Comment:
Won't work if the user isn't in that particular directory - how to change it depends on how we want it to be installed for users. Either remove the `./` or you'd need to get the directory name of the `LINKS` script
```suggestion
./LINKS-make LINKS $f $s $m $d $k $t $o $e $l $a $z $b $r $p $x $v
```